dirty_cat/test/test_super_vectorizer.py

```diff
@@ -434,6 +434,40 @@ def test_fit_transform_equiv() -> None:
     assert np.allclose(enc1_x2, enc2_x2, rtol=0, atol=0, equal_nan=True)
 
 
+def test_passthrough():
+    """
+    Tests that when passed no encoders, the SuperVectorizer
+    returns the dataset as-is.
+    """
+
+    def is_equal(elements: Tuple[Any, Any]) -> bool:
```

Comment:
In general, I prefer avoid defining functions inside function (they cannot be reused, they cannot be tested, they cannot be pickled, hence they can lead to trouble)